policies/WindowsTerminal.admx

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--  (c) 2024 Microsoft Corporation  -->
+<policyDefinitions xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" revision="1.0" schemaVersion="1.0" xmlns="http://schemas.microsoft.com/GroupPolicy/2006/07/PolicyDefinitions">
+  <policyNamespaces>
+    <target prefix="terminal" namespace="Microsoft.Policies.WindowsTerminal" />
+    <using prefix="windows" namespace="Microsoft.Policies.Windows" />
+  </policyNamespaces>
+  <resources minRequiredRevision="1.0" />
+  <supportedOn>
+    <definitions>
+      <definition name="SUPPORTED_WindowsTerminal_1_21" displayName="$(string.SUPPORTED_WindowsTerminal_1_21)" />
+    </definitions>
+  </supportedOn>
+  <categories>
+    <category name="WindowsTerminal" displayName="$(string.WindowsTerminal)">
+      <parentCategory ref="windows:WindowsComponents" />
+    </category>
+  </categories>
+  <policies>
+    <policy name="DisabledProfileSources" class="Both" displayName="$(string.DisabledProfileSources)" explainText="$(string.DisabledProfileSourcesText)" presentation="$(presentation.DisabledProfileSources)" key="Software\Policies\Microsoft\Windows\Terminal">
+      <parentCategory ref="WindowsTerminal" />
+      <supportedOn ref="terminal:SUPPORTED_WindowsTerminal_1_21" />

```

Comment:
```suggestion
      <supportedOn ref="SUPPORTED_WindowsTerminal_1_21" />
```

Is unique enough.

Reply:
1.21 is the current stable and I am sure it does not support the policy. Shouldn't it be 1.22 or 1.23?

Reply:
We intend to backport this change to 1.21 along with other bug fixes to help a customer. 🙂

Reply:
But then please provide an exact version to avoid confusion: 1.21.<revision>

Reply:
The .adml contains:
```
<string id="SUPPORTED_WindowsTerminal_1_21">At least Windows Terminal 1.21</string>
```

Is that not enough? I know that the `supportedOn` element can contain a version, but I only saw it being used on the OS version, not on package versions...